appyters/dimensionality_reduction_visualization/appyter.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://raw.githubusercontent.com/MaayanLab/appyter-catalog/main/schema/appyter-validator.json",
   "name": "dimensionality_reduction_visualization",
   "title": "PCA, t-SNE, UMAP Appyter",
-  "version": "0.0.9",
+  "version": "0.1.0",
```

Comment:
Since there are no new features, you're just patching existing features. This should be `0.0.10`